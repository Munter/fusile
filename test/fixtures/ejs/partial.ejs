<p>here's a partial my friends:</p>
<% include includeme %>