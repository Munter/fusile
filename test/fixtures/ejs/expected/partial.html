<p>here's a partial my friends:</p>
<p>look ma i made it to the partial!</p>